Dockerfile

```diff
@@ -1,31 +1,33 @@
-FROM centos:6
+FROM ruby:2.4.5-slim-stretch
 
 # Match the jenkins uid/gid on the host (504)
 RUN groupadd -r gibct && \
-      useradd -r -g gibct gibct
-
-RUN yum install -y git make gcc-c++ openssl-devel readline-devel zlib-devel sqlite-devel postgresql-devel socat timeout epel-release nc
-
-# Install Red Hat SCI library for ruby packages
-RUN yum install -y centos-release-scl-rh
-RUN yum install -y rh-ruby23 rh-ruby23-ruby-devel rh-ruby23-rubygems
-RUN echo "source /opt/rh/rh-ruby23/enable" > /etc/profile.d/rh-ruby23.sh
-RUN ["/bin/bash", "--login", "-c", "gem install --no-doc bundler"]
+    useradd -r -g gibct gibct && \
```

Comment:
Please review this list and only add what is needed for gids.

Reply:
- `build-essential` normal package to install on Debian to allow stuff to happen
- `git` needed to install sniffybara gem and others with specific sha refs
- `curl` needed to install [phantomjs](https://gist.github.com/julionc/7476620) (not the exact same process as listed)
- `clamav` needed to run [freshclam](https://wiki.archlinux.org/index.php/ClamAV)
- `libpq-dev` need to install gem `pg`
- `libgmp-dev` not needed, Multiprecision arithmetic library
- `imagemagick` not needed, pdf related
- `pdftk` not needed, pdf related
- `poppler-utils` not needed, pdf related